Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -102,15 +101,31 @@ public ConceptEntry getWordnetConceptEntry(String wordnetId) throws LuceneExcept
      * java.lang.String, java.lang.String)
      */
     @Override
-    public ConceptEntry[] getConceptListEntriesForWord(String word, String pos, String conceptList)
+    public ConceptEntry[] getConceptListEntriesForWordPOSConceptList(String word, String pos, String conceptList)
```

Comment:
given that the method is already called get**ConceptListEntries**ForWord... it seems doppeltgemoppelt (doublemoppeled) to attach ConceptList at the end of the method.